tests/integration/cmor/_fixes/cmip6/test_cesm2.py

```diff
@@ -61,7 +61,6 @@ def test_get_cl_fix():
 )
 
 
-@pytest.mark.sequential
 @pytest.mark.skipif(
     sys.version_info < (3, 7, 6), reason="requires python3.7.6 or newer"
 )
```

Comment:
Why are you dropping the `sequential` mark? Also, would it make sense to remove the following `skipif` since we're not supporting Python 3.7 anymore?

Reply:
It was removed in #1819, but these entries were forgotten. Using
https://github.com/ESMValGroup/ESMValCore/blob/edb1291944ded032fa2608a894c60ce70ca49304/pyproject.toml#L151
makes using undefined markers an error.

Reply:
Removed in e70ffd2